terraform/workspace-variables/qa.tfvars

```diff
@@ -13,6 +13,8 @@ distribution_list = {
   }
 }
 
+enable_cloudfront_compress = "true"
```

Comment:
We could make `true` default and remove it from here
Make sure you use `true` as boolean and not `"true"` as string, it's cleaner